opentelemetry-api/src/opentelemetry/environment_variables.py

```diff
@@ -36,3 +36,14 @@
 """
 .. envvar:: OTEL_PYTHON_TRACER_PROVIDER
 """
+
+OTEL_PYTHON_METER_PROVIDER = "OTEL_PYTHON_METER_PROVIDER"
+"""
+.. envvar:: OTEL_PYTHON_METER_PROVIDER
+"""
+
+OTEL_METRICS_EXPORTER = "OTEL_METRICS_EXPORTER"
```

Comment:
These are the only real positives from the publicApiCheck. I think it is OK to add these though because

1. They are unlikely to change
2. If we do change them, can just leave these old variables hanging around as deprecated for backward compat

Reply:
I can also add underscores. Wdyt?

Reply:
Add an underscore seems good to me.

Reply:
PTAL at https://github.com/open-telemetry/opentelemetry-python/pull/2255. I will cherry-pick that after merging.

Reply:
Fixed